docs/pages/eas/workflows/syntax.mdx

```diff
@@ -480,7 +489,7 @@ jobs:
 
 ### `jobs.<job_id>.steps.<step>.name`
 
-The name of the step, which is displayed in the job's logs.
+The human-friendly name of the step, which is displayed in the job's logs. When not provided, the step name is the command to run.
```

Comment:
```suggestion
The human-friendly name of the step, which is displayed in the job's logs. When not provided, the step name is the command to run.
```

The last part is a bit confusing. Looking at the example below, does that mean it will run `steps.name`?

Reply:
Just edited this to be more clear:

"The human-friendly name of the step, which is displayed in the job's logs. When a step's name is not provided, the `run` command is used as the step name."